antismash/modules/thiopeptides/specific_analysis.py

```diff
@@ -37,7 +37,7 @@ def to_json(self) -> Dict[str, Any]:
                                    for key, features in self.cds_features.items()}
         return {"record_id": self.record_id,
                 "schema_version": ThioResults.schema_version,
-                "clusters with motifs": [cluster.get_cluster_number() for cluster in self.clusters_with_motifs],
+                "clusters with motifs": [cluster.get_protocluster_number() for cluster in self.clusters_with_motifs],
```

Comment:
Not sure if we want to change the result object JSON to "protoclusters with motifs" here.